pkg/storage/system_intake_relation.go

```diff
@@ -1,7 +1,73 @@
 package storage
 
 import (
-	_ "embed" // embed is used to embed the SQL file content
+	"context"
+	"errors"
+
+	"github.com/google/uuid"
+	"github.com/jmoiron/sqlx"
+	"go.uber.org/zap"
+
+	"github.com/cmsgov/easi-app/pkg/appcontext"
+	"github.com/cmsgov/easi-app/pkg/models"
 )
 
 // TODO Implement store methods that have to deal with setting System Intake linking/relation data
+func (s *Store) LinkSystemIntakeContractNumbers(ctx context.Context, tx *sqlx.Tx, systemIntakeID uuid.UUID, contractNumbers []string) error {
+	if systemIntakeID == uuid.Nil {
+		return errors.New("unexpected nil system intake ID when linking system intake to contract number")
+	}
+
+	deleteStatement := `
```

Comment:
Could you please embed these queries in the sqlqueries package please?

Reply:
Yes! Done! Thanks Steven 😄